Maven testsuite doesn't pass on non-US environement

Steps
1. cd testsuite/testcases
2. ../../mvnw test -Dtestcases=java

Expected:
test passes

Actual:
```
Test failed, unexpected jflex output: Difference in line 7
expected :[Constructing NFA : 1,010 states in NFA]
but got  :[Constructing NFA : 1.010 states in NFA]
JFlex output: Reading "src/test/cases/java/java.flex"

Warning in file "src/test/cases/java/java.flex" (line 340):
".|\n" does not match all characters, because "." excludes all Unicode newline chars - use "[^]" instead
.|\n                             { throw new RuntimeException("Illegal character \""+yytext()+
^
Constructing NFA : 1.010 states in NFA
Converting NFA to DFA :
................................................................................................................................................................................................................................................................................................................................................................................................................................................................
454 states before minimization, 426 states in minimized DFA
Writing code to "src/test/cases/java/Java.java"

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.207 s
[INFO] Finished at: 2020-05-12T21:57:52+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal de.jflex:jflex-testsuite-maven-plugin:1.9.0-SNAPSHOT:run-test-suite (default) on project jflex-testsuite-testcases: Failed to execute test suite: Test [src/test/cases/java/java.test] failed: Unexpected jflex output:
[ERROR] Difference in line 7
[ERROR] expected :[Constructing NFA : 1,010 states in NFA]
[ERROR] but got  :[Constructing NFA : 1.010 states in NFA]
[ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal de.jflex:jflex-testsuite-maven-plugin:1.9.0-SNAPSHOT:run-test-suite (default) on project jflex-testsuite-testcases: Failed to execute test suite: Test [src/test/cases/java/java.test] failed
```

--- a/testsuite/jflex-testsuite-maven-plugin/src/main/java/jflex/maven/plugin/testsuite/TestsuiteUtils.java
+++ b/testsuite/jflex-testsuite-maven-plugin/src/main/java/jflex/maven/plugin/testsuite/TestsuiteUtils.java
@@ -5,6 +5,7 @@ import java.io.FileReader;
 import java.io.FilenameFilter;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Locale;
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugin.MojoFailureException;
 
@@ -70,6 +71,7 @@ public class TestsuiteUtils {
    */
   public static boolean runTests(List<File> tests, File jflexUberJar)
       throws TestFailException, MojoExecutionException, MojoFailureException {
+    Locale.setDefault(Locale.US);
     int successCount = 0;
     int totalCount = 0;
 

